bambi/terms/utils.py

```diff
@@ -42,6 +42,16 @@ def is_truncated_response(term):
     return is_call_of_kind(component, "truncated")
 
 
+def is_constrained_response(term):
```

Comment:
I like how each function  "is...truncated, constrained, and weighted" are explicitly defined, but the only difference between the them is the final return `is_call_of_kind(component, "<response>")`. Is there any value in moving these into a single function other than to delete duplicate code?